presto-main/src/main/java/com/facebook/presto/cost/StatsCalculators.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cost;
+
+import static java.util.Objects.requireNonNull;
+
+public class StatsCalculators
```

Comment:
From what i understand by introducing this class we are trying to avoid annotating StatsCalculator. We would have to annotate them again to make this class injectable.